### PR TITLE
[WIP] Add dirtiness flag saving to prototype

### PR DESCRIPTION
**Pull request by NoahCarnahan**
@czue @snopoke cc @dannyroberts 
First steps to generalizing saving of dirtiness flags. Probably easier to review with `w=1`.

### CODE REVIEW

**Comment by czue** [comment]
:+1: from a quick look - will let @snopoke review

File: prototype/mobile_endpoint/backends/sql/dao.py [hunk 1]
```diff
@@ -68,7 +70,7 @@ def get_case(self, id, lock=False):
             None, CaseData.query.get(id)
 
     def case_exists(self, id):
-        return CaseData.query.filter_by(id=id).exists()
+        return CaseData.query.session.query(exists().where(CaseData.id == id)).scalar()
```
Comment (snopoke) [inline prototype/mobile_endpoint/backends/sql/dao.py:73]: nice

**Comment by snopoke** [comment]
Seems I may have miscommunicated what I intended. I don't think we need to worry about storing OwnershipCleanliness data in mongo / couch. Sorry if that wasn't clear. I think it makes sense to just keep these in postgres. I know that mixes the DB's a bit and perhaps then we should just not store them at all for the benchmarks?

Let me know what you think.

**Comment by NoahCarnahan** [comment]
Ah I see. I guess the main downside of storing them in postgres for all backends is that it would be hard to tell if storing OwnershipCleanliness is the bottle neck. On the other hand, not storing them at all makes comparison with the current receiver more difficult. I think I'm in favor of storing them in postgres for all backends at the moment.

**Pull request closed without merging**